actions/dhcp/README.md

```diff
@@ -0,0 +1,9 @@
+---
+slug: Re DHCP
+name: Re DHCP
+tags: disk
+maintainers: Marky Jackson <marky.r.jackson@gmail.com>
+description: "This action is for release and renew of DHCP
+version: v1.0.0
+createdAt: "2021-03-03T12:41:45.14Z"
+---
```

Comment:
The body of the README will get rendered by ArtifactHub, it is important to have a descriptive and comprehensive one because it acts as documentation. A good structure can be:

```
## What this action does

## Why and when it should be used

## Examples

here you can place snippets of yaml explaining the various bit you can pass to the action itself

Reply:
Ok, I will add those bits